include/shark/LinAlg/BLAS/detail/functional.hpp

```diff
@@ -105,6 +105,16 @@ struct scalar_tanh{
 };
 
 template<class T>
+struct scalar_acos{
```

Comment:
all changes to LinAlg have to go a different route because git can not really handle the way we split the repos. I will take care of that